### PR TITLE
Adds the preflight release_stage

### PR DESCRIPTION
**Pull request by Azurelol**
## Description

Adds support for the preflight release stage.

## Changes
* ![ADD] Support for the newer `preflight` release stage, as a task
* ![UPDATE] `atlas-version` to `1.1+`, as it includes the newer release stage and wdk strategy.
 
[NEW]:      https://resources.atlas.wooga.com/icons/icon_new.svg "New"
[ADD]:      https://resources.atlas.wooga.com/icons/icon_add.svg "Add"
[IMPROVE]:  https://resources.atlas.wooga.com/icons/icon_improve.svg "Improve"
[CHANGE]:   https://resources.atlas.wooga.com/icons/icon_change.svg "Change"
[FIX]:      https://resources.atlas.wooga.com/icons/icon_fix.svg "Fix"
[UPDATE]:   https://resources.atlas.wooga.com/icons/icon_update.svg "Update"
[BREAK]:    https://resources.atlas.wooga.com/icons/icon_break.svg "Remove"
[REMOVE]:   https://resources.atlas.wooga.com/icons/icon_remove.svg "Remove"
[IOS]:      https://resources.atlas.wooga.com/icons/icon_iOS.svg "iOS"
[ANDROID]:  https://resources.atlas.wooga.com/icons/icon_android.svg "Android"
[WEBGL]:    https://resources.atlas.wooga.com/icons/icon_webGL.svg "WebGL"
[GRADLE]:   https://resources.atlas.wooga.com/icons/icon_gradle.svg "GRADLE"
[UNITY]:    https://resources.atlas.wooga.com/icons/icon_unity.svg "Unity"
[LINUX]:    https://resources.atlas.wooga.com/icons/icon_linux.svg "Linux"
[WIN]:      https://resources.atlas.wooga.com/icons/icon_windows.svg "Windows"
[MACOS]:    https://resources.atlas.wooga.com/icons/icon_iOS.svg "macOS"

### CODE REVIEW

File: src/integrationTest/groovy/wooga/gradle/release/ReleasePluginIntegrationSpec.groovy [hunk 1]
```diff
@@ -440,6 +440,7 @@ class ReleasePluginIntegrationSpec extends IntegrationSpec {
         "final"   | false
         "release" | false
         "rc"      | true
+        "pre"     | true
```
Comment (Larusso) [inline src/integrationTest/groovy/wooga/gradle/release/ReleasePluginIntegrationSpec.groovy:443]: I thought we call it `prefligh` everywhere

**Review by Larusso — CHANGES_REQUESTED** [review]
Can we also set the version scheme to `wdk` by default? So we don't need to change this in every wdk. We need to remove the property anyways in all repos we want to integrate this but just in case,

File: src/integrationTest/groovy/wooga/gradle/release/ReleasePluginIntegrationSpec.groovy (hunk 1 above)
Comment (Azurelol) [reply src/integrationTest/groovy/wooga/gradle/release/ReleasePluginIntegrationSpec.groovy:443]: I am adding it onto atlas-version.

File: src/main/groovy/wooga/gradle/release/ReleasePlugin.groovy [hunk 2]
```diff
@@ -124,18 +128,23 @@ class ReleasePlugin implements Plugin<Project> {
...
+//                    break
+//                case VERSION_SCHEME_SEMVER_1:
+//                    versionExtension.versionScheme(VersionScheme.semver)
+//                    break
+//                case VERSION_SCHEME_WDK:
+//                    versionExtension.versionScheme(VersionScheme.wdk)
+//                    break
+//            }
+//        }
+        versionExtension.versionScheme.convention(VersionScheme.wdk)
```
Comment (Larusso) [inline src/main/groovy/wooga/gradle/release/ReleasePlugin.groovy:147]: No I mean adjusting the `VERSION_SCHEME_DEFAULT` to point to `wdk`

**Review by Larusso — CHANGES_REQUESTED** [review]

File: src/main/groovy/wooga/gradle/release/ReleasePlugin.groovy (hunk 2 above)
Comment (Larusso) [reply src/main/groovy/wooga/gradle/release/ReleasePlugin.groovy:147]: Or wait. Scrap this. Leave the old way for now and we deal with this in the WDKs for now.

**Review by Larusso — APPROVED** [review]

**Pull request merged**